lesson_02_homework.rb

```diff
@@ -26,21 +29,35 @@ def full_name(f, l)
   puts f + l
 end
 
-puts full_nam("bubba", "chuck")
+puts full_name("bubba", "chuck")
 
 
 # Coding
 
 
 ## Enter your code for your 'add' method below
+def add(num1, num2)
+  puts "Total numer is #{add = num1 + num2}"
+end
 
+puts add(34, 45)
 
 
 
 ## Enter the code for you 'join_strings' method below
+def favorite_words(string1, string2)
+  puts "'#{string1+ " "+ string2}'"
+end
 
-
-
+puts favorite_words("Good", "Luck")
 
 
 ## Enter your 'old_enough_to_vote?' method below
+voter = 72
+if voter >= 18 && voter <= 65 
+  puts "You are good to vote"
```

Comment:
🔨  @Alisher778 using `puts` in the method body makes the method less flexible - if you remove the `puts` you will achieve the same result without forcing the method to print information out to the console.